en/assembly/quick_start_pixhawk4.md

```diff
@@ -26,33 +30,29 @@ Pixhawk 4 should be mounted on the frame using vibration-damping foam pads (incl
 
 ## GPS + Compass + Buzzer + Safety Switch + LED
 
-Attach the provided GPS with integrated compass, safety switch, buzzer and LED to the GPS port.
+Attach the provided GPS with integrated compass, safety switch, buzzer and LED to the **GPS MODULE** port.
 
-> **Note** The diagram shows a GPS with an integrated compass.  The GPS/Compass should be mounted on the frame as far away from 
-  other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other 
-  electronics will reduce interference).
+The GPS/Compass should be mounted on the frame as far away from other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other electronics will reduce interference).
 
-![Connect compass/GPS to Pixhawk 4](../../images/pixhawk4_compass_gps.png)
+![Connect compass/GPS to Pixhawk 4](../../assets/flight_controller/pixhawk4/pixhawk4_compass_gps.jpg) 
 
-When GPS module with integrated safety switch is connected to Pixhawk 4, by default the safety is enabled. This means the autopilot will not let you arm the vehicle. To arm the vehicle, you have to press and hold the safety switch for 1 sec to disable safety. Similarly, if the vehicle is armed and something goes wrong such that you are unable to disarm the vehicle from QGC or RC, you can press the safety switch again to disarm, enabling safety.
+> **Note** The GPS module's integrated safety switch is enabled *by default* (when enabled, PX4 will not let you arm the vehicle). To disable the safety press and hold the safety switch for 1 second. You can press the safety switch again to enable safety and disarm the vehicle (this can be useful if, for whatever reason, you are unable to disarm the vehicle from your remote control or ground station).
 
 
 
 ## Power

```

Comment:
@barzanisar This whole power section needs a lot of work - it is confusing and mixes information that it is not clear whether should be should not be here or now (e.g. motor signal pins). 

It needs to cover these things:
1. What is a power management board
1. The fact that flight controller (+ receivers), and the flight controller servo output rail must be powered separately
1. How to power the flight controller boards + receiver:
   - with a single supply using the POWER1 port.
   - With triple redundancy. Does not have to cover this case  in detail but should link to info on the topic and/or tips. (ie. do you need 3 power management boards?
1. How to power output rails
1. Diagram showing single and triple-reduntant power setup would be lovely :-)
1. Also needs to answer how to make the connection to the power board. It just looks like a PCB here, so I guess you solder? If so that is worth saying, because I would hope that it would have a snap connector and you could just attach a wire. 

You have a lot of that but it is a bit jumbled.

This could start with a sentence like:

> *Pixhawk 4* has 3 power inputs (x,y, z) that can be used to power the autopilot itself (and connected receivers) and X, Y inputs that can be used to power servos connected to the output rails. The autopilot can be powered through any of the three inputs or via all of them for triple redundancy.
> 
> The simplest setup is shown below ....




is powered from a LiPo battery via a *Power Management Board* (PMB). This board provides separate supply to the flight controller (including connected receivers) and to power the output rail for motors/servos.